package/yast2-ruby-bindings.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Dec  1 16:06:11 UTC 2015 - jreidinger@suse.com
+
+- revert last change as it break using cell for quering table cell
```

Comment:
> Reverted the last change because it broke updating the table in Service Manager
> (because the distinction between :cell and :Cell was lost; bnc#956380)